django/db/models/constraints.py

```diff
@@ -26,6 +27,15 @@ def clone(self):
         _, args, kwargs = self.deconstruct()
         return self.__class__(*args, **kwargs)
 
+    def set_name_with_dict(self, model):
```

Comment:
Making a separate method just for formatting `name` seems an unnecessary abstraction to me, what do you think?

Reply:
You are right, I thought that I can handle it into `__init__`  but we don't have `model` unless the method is call by `ModelBase._prepare`